dapps/marketplace/src/components/DesktopUserActivation.js

```diff
@@ -293,6 +293,7 @@ class UserActivation extends Component {
                 address
               </fbt>
             </div>
+            <div className="modal-spacer" />
```

Comment:
Can we avoid layout related divs like this by using :before and :after css pseudo classes?

Reply:
Pseudo classes cannot be used here. That class has just `flex: auto`. I added that as a spacer so that the action buttons will always stick to the bottom of the screen on mobile. Perhaps I will try using `position: fixed` or something else.

Reply:
If the parent is `display: flex` you could probably say `margin-top: auto` on the `actions` div for mobile?

Reply:
Yep, It works too.